Polyhedron/demo/Polyhedron/Statistics_on_item_dialog.ui

```diff
@@ -63,6 +49,44 @@
      </item>
     </layout>
    </item>
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
+     <item>
+      <widget class="QPushButton" name="exportButton">
+       <property name="text">
+        <string>Export</string>
```

Comment:
The text should be "Export...", and even better: "&Export..." (that adds a shortcut <kbd>Alt+E</kbd>).

https://ux.stackexchange.com/q/9544